### ISSUE TITLE
No snow buildup when snow is caused by Serene Seasons

### ISSUE THREAD

**Issue by Moleculor** [Snownee/SnowRealMagic#100]
Minecraft 1.16.4
```
	Mod List: 
		forge-1.16.4-35.1.15-client.jar                   |Minecraft                     |minecraft                     |1.16.4              |DONE      |NOSIGNATURE
		SereneSeasons-1.16.4-4.0.0.82-universal.jar       |Serene Seasons                |sereneseasons                 |1.16.4-4.0.0.76     |DONE      |NOSIGNATURE
		forge-1.16.4-35.1.15-universal.jar                |Forge                         |forge                         |35.1.15             |DONE      |22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		SnowRealMagic-1.16.3-2.2.2.jar                    |Snow! Real Magic!             |snowrealmagic                 |2.2.2               |DONE      |NOSIGNATURE
		Kiwi-1.16.3-3.4.1.jar                             |Kiwi                          |kiwi                          |3.4.1               |DONE      |NOSIGNATURE
```

When in a snowy biome, and `snowAccumulationDuringSnowfall = true` is set to true (non-default), snow will accumulate as expected, including occupying tall grass blocks, etc. 

When in a cold biome during Early Spring in [Serene Seasons](https://www.curseforge.com/minecraft/mc-mods/serene-seasons), rainfall is snowfall, not rain. This causes vanilla-style snow to accumulate, but no SnowRealMagic style buildup and no snow in tall grass blocks, despite the buildup still occurring in other biomes. 

This is after ten or twenty minutes of "rain" in Serene Seasons in `minecraft:mountains` at a low altitude:
![image](https://user-images.githubusercontent.com/3865976/103179785-1cf26600-4855-11eb-98f4-dbf2c949c294.png)

After five or ten minutes of snowfall (in Serene Seasons) in `minecraft:snowy_taiga` where snow would fall during normal vanilla Minecraft weather:
![image](https://user-images.githubusercontent.com/3865976/103179802-4e6b3180-4855-11eb-895b-dd45ff4bdcbc.png)

(There's also a [Better Weather](https://www.curseforge.com/minecraft/mc-mods/better-weather) mod that adds seasons, too, if it matters.)

**Comment by Snownee** [comment]
Duplicate of #68

**Comment by Moleculor** [comment]
Following that chain, the remark is given:

> Glitchfiend/SereneSeasons#247
> I assume it will be fixed once this get fixed. If not you can report again then

That was fixed 16 days ago, and unless the latest beta version they released a day ago still didn't contain the fix, the fix for Glitchfiend/SereneSeasons#247 is in the version I tested and it still doesn't work.
